common/actions/config.js

```diff
@@ -1,12 +1,18 @@
 // @flow
+import { setLanguage } from 'translations';
 export const CONFIG_LANGUAGE_CHANGE = 'CONFIG_LANGUAGE_CHANGE';
 export const CONFIG_NODE_CHANGE = 'CONFIG_NODE_CHANGE';
 
+export const CHANGE_LANGUAGE = (value: any) => {
+    setLanguage(value.sign);
+    return {
+        type: CONFIG_LANGUAGE_CHANGE,
+        value
+    };
+};
 
-export const CHANGE_LANGUAGE = (value: any) => Object({
-    type: CONFIG_LANGUAGE_CHANGE, value
-})
-
-export const CHANGE_NODE = (value: any) => Object({
-    type: CONFIG_NODE_CHANGE, value
-})
+export const CHANGE_NODE = (value: any) =>
```

Comment:
@dternyak yeah i am trying to not change files you could be touching, this is a prettier thing. One time formatting war :D